tspng/extraction.py

```diff
@@ -25,7 +26,7 @@ def extract_from_file(path, mime_type=MIME_TYPE):
     dict=json.loads(meta[mime_type])
     return dict
 
-def extract_from_files(paths, mime_type=MIME_TYPE):
+def extract_from_files(paths: List, mime_type: str=MIME_TYPE):
```

Comment:
A type for the return value should be added here, too. I think it can actually be `Dict[Dict]`, but please check on this.

Reply:
Change made, Dict[Dict] did not work so for right now it is Dict